# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1533884913 -7200
#      Fri Aug 10 09:08:33 2018 +0200
# Branch stable
# Node ID 66caa6dc6701f78c87414870e49a944b82cb90bc
# Parent  2d5bd2cce0ddd581d2887b108572dd31d7b20a31
* m4/octave_blas_f77_func.m4: Restore 64-bit integer BLAS detection (bug #53853).

* m4/octave_blas_f77_func.m4: The unnecessary else-branch was introduced in
  cset 171d90967f16 to fix bug #53853 and fails this test under any circumstance
  and makes this last if-branch pointless.

diff --git a/m4/octave_blas_f77_func.m4 b/m4/octave_blas_f77_func.m4
--- a/m4/octave_blas_f77_func.m4
+++ b/m4/octave_blas_f77_func.m4
@@ -193,18 +193,16 @@ c was compiled with 4-byte integers.
 
 *     print *, a(1), b(1), d
 
 c Success (0 exit status) means we detected BLAS compiled with
 c 8-byte integers.
 
       if (d .eq. 1.0) then
         stop 1
-      else
-        stop 1
       endif
 
        ]]),[ax_blas_integer_size=8],
         [ax_blas_integer_size=4])
         AC_MSG_RESULT([$ax_blas_integer_size])
 
         AC_LANG_POP(Fortran 77)
 
